x/machine/keeper/msg_server_attest_machine.go

```diff
@@ -44,16 +50,22 @@ func validateIssuerLiquid(issuerLiquid string) bool {
 	return isValidLiquidKey
 }
 
-func (k msgServer) reissueMachine(machine *types.Machine) error {
+func (k msgServer) issueMachineNFT(machine *types.Machine) error {
```

Comment:
we have reissue in our proto:
```
proto/planetmintgo/machine/machine.proto:11:  bool reissue = 4;
```

Reply:
That's different. The reissue property is about the character of the token. the function is not about reissuance but issuing an NFT or a more generic token https://help.blockstream.com/hc/en-us/articles/900005127583-How-do-I-issue-an-asset-on-Liquid-